### PR TITLE
Add package nish

### PR DESCRIPTION
**Pull request by owlinux1000**

### CODE REVIEW

**Comment by Yardanico** [comment]
@owlinux1000 it seems that you've opened a PR two times - please close either this one or https://github.com/nim-lang/packages/pull/933 :)

**Comment by owlinux1000** [comment]
I'm sorry for PR duplicated. 
This is the first time using nimble publish.
I'll close #933!

thanks

File: packages.json [hunk 1]
```diff
@@ -9295,7 +9295,7 @@
       "pqcrypto",
       "signing"
     ],
-    "description": "SPHINCS⁺ stateless hash-based signature scheme",
+    "description": "SPHINCS\u207A stateless hash-based signature scheme",
```
Comment (treeform) [inline packages.json]: I think unicode crap happened?

Comment (dom96) [reply packages.json]: Not @owlinux1000's fault and it doesn't matter too much IMO.

**Pull request merged**